spec/support/bedford_pals.rb

```diff
@@ -0,0 +1,32 @@
+# This class defines a set of users, schools, homerooms and students
+# that can be used for testing Bedford-style authorization paths.
+class BedfordTestPals
```

Comment:
Minimal for now, meant to facilitate local testing against Bedford-style logins.